_core/smart/tutorial-counter.md

```diff
@@ -27,13 +27,14 @@ Before you get started, you should complete the [Hello World tutorial](tutorial.
 
 In this step, you initialize a starter project with additional counter tutorial files:
 
-Using your terminal, run the following command:
```

Comment:
the link above in the reqs refers to the "Hello World tutorial" it is renamed to "Get Started"

Reply:
I changed it back to Hello World; we are using "Get Started" for another area, after all.